### ISSUE TITLE
Responsiveness needs tidying up

### ISSUE THREAD

**Issue by sarahwelton** [LDMW/cms#505]
**We need the site to be fluid!** 
This should not be happening - the site should be fluid and responsive: 
<img src="https://user-images.githubusercontent.com/26305143/30848155-1e5be63c-a296-11e7-9bb9-73871cc04ee3.png" width=300px />

**1020px width:** 
<img width="1020" alt="screen shot 2017-09-26 at 08 58 49" src="https://user-images.githubusercontent.com/26305143/30849086-134ef27c-a299-11e7-994e-333c858b9f6f.png">
<img width="1020" alt="screen shot 2017-09-26 at 08 59 19" src="https://user-images.githubusercontent.com/26305143/30849085-134dfbba-a299-11e7-8af9-eca76516df0f.png">

**why is the introduction line breaking here?** 
<img width="1036" alt="screen shot 2017-09-26 at 09 03 38" src="https://user-images.githubusercontent.com/26305143/30849268-ace874a8-a299-11e7-984c-eb8ee3af3943.png">

**There doesn't need to be so much white space around the edge here**
- on wide screens, yes, we need the "reading width" of a line to be limited. But the width of the content column should remain the same size and the white space on each side should reduce until there is a small margin on each side, eg 20px. Then the content column can start to reduce.
<img width="1190" alt="screen shot 2017-09-26 at 09 06 20" src="https://user-images.githubusercontent.com/26305143/30849389-1986b2dc-a29a-11e7-9860-454d31d22da9.png">

**Comment by iteles** [comment]
@ellemindwave I've added a high level estimate here but we'll need the acceptance criteria before for this before we can confirm fully. 

We test this on break points that are known break points for various phones and tablets as well as desktops, but we don't test for every single breakpoint between desktop and phone as these are not typically used. We will need to establish which ones we need for the acceptance criteria.

**Comment by reddog** [comment]
(just a note to any future readers - it's showing that @sarahwelton assigned @Danwhy, @shouston3 & @Cleop and then I unassigned them - I didn't *knowingly* do this!)

**Comment by reddog** [comment]
@iteles Thanks for your comment

> We test this on break points that are known break points for various phones and tablets as well as desktops, 

Could you let us know what those break points are?  We could then ensure we are testing against those break points and/or specifying others where required.

**Comment by Cleop** [comment]
@reddog we use the devices provided in dev tools to test, those are currently:
<img src="https://user-images.githubusercontent.com/16775804/31071849-81cc44c0-a75d-11e7-9774-3ba05a95afa0.png" width=150px />
We tend to use iPhone 5C as a baseline as it's the narrowest for mobiles and then we also check iPad for tablet. Obviously the focus on different devices may vary from project to project so sometimes we'll focus on one area more than another depending on your needs but this is our starting point. It's easier for us to fix things when we can all replicate the issue on a given device and we know an issue is likely to arise for a number of users rather than if a user randomly happens to contort their browser to a certain shape. We also test on our own devices as dev tools isn't always 100% accurate and this will just be what the dev team has available to them!

As a **starting point** for **acceptance criteria** I have found one of the bugs you found on Ipad Pro view:
- [ ] Add more padding underneath text before the button
![image](https://user-images.githubusercontent.com/16775804/31072127-9d2fc902-a75e-11e7-81a2-6332a1b67dcd.png)

- [ ] On iPad, previous question is not on the same line as its arrow icon, make it inline:
<img src="https://user-images.githubusercontent.com/16775804/31072469-fd8601b2-a75f-11e7-80c2-47102489bcb3.png" width=250px />

But @ellemindwave @sarahwelton if you could give directions for finding and then proposed solutions for the other screenshots above that would be great.

**Comment by ellemindwave** [comment]
I have added an issue that will hopefully help with Sarah's first screenshot above #528

**Comment by ellemindwave** [comment]
On ipad, landscape, chrome

 - [ ] Please can the logos at the bottom of the screen remain on one line and just reduce in size on smaller screens? or split so they are 3 and then 2 on another line? 
It looks quite odd having the councils logo on its own on the bottom line

![screen shot 2017-10-03 at 16 30 36](https://user-images.githubusercontent.com/26304634/31133748-3c9b9e72-a858-11e7-95d2-35bb42763a06.png)

**Comment by ellemindwave** [comment]
ipad, portrait, chrome

- [ ] top navbar is too fat and becomes too close to the header

![screen shot 2017-10-03 at 16 33 36](https://user-images.githubusercontent.com/26304634/31133941-c0dcbfc2-a858-11e7-834a-bb69808917f6.png)

**Comment by ellemindwave** [comment]
ipad, portrait, chrome

- [ ] two-lined 'Sleep deprived'  title in the nav bar needs more spacing, looks crushed

![screen shot 2017-10-03 at 16 35 14](https://user-images.githubusercontent.com/26304634/31134036-fce1f410-a858-11e7-953f-80938b1bb711.png)

**Comment by ellemindwave** [comment]
ipad, portrait, chrome & Safari

Please can ipad portrait always be mobile view

![screen shot 2017-10-03 at 16 37 16](https://user-images.githubusercontent.com/26304634/31134164-3ff24ee4-a859-11e7-8af6-15f4d06f8d37.png)

**Comment by ellemindwave** [comment]
ipad, landscape, chrome

- [ ] buttons in beta banner are one on top of the other, therefore they are hiding the nav bar beneath
![screen shot 2017-10-03 at 16 38 43](https://user-images.githubusercontent.com/26304634/31134238-7205e5d0-a859-11e7-982a-ae10eff0411b.png)

**Comment by ellemindwave** [comment]
iPhone 6, landscape, chrome

-  [ ] Please can **mobiles** (iphone 5, phone 6, iphone 6 plus) on landscape all just automatically be in mobile view? This will solve lots of responsiveness issues I think :). Switch to burger menu. Check when break point for topic cards stacking.

![screen shot 2017-10-03 at 16 41 42](https://user-images.githubusercontent.com/26304634/31134396-ddc8a0c8-a859-11e7-9c88-90b07dbec4c9.png)

**Comment by ellemindwave** [comment]
iPad pro, portrait, chrome

- [ ] beta banner on two lines, hides nav bar below
- [ ] iPad pro portrait, always be mobile view

![screen shot 2017-10-03 at 16 45 00](https://user-images.githubusercontent.com/26304634/31134679-bd98753e-a85a-11e7-8c11-bd0203e164a7.png)

**Comment by ellemindwave** [comment]
iPhone 7, Safari, iOS 11

The text disappears off the edge of the screen , as do the buttons, of topics cubes on homepage

![image uploaded from ios 26](https://user-images.githubusercontent.com/26304634/31186628-13d38444-a927-11e7-9a86-87db6a128d3c.jpg)

**Comment by katbow** [comment]
Some of the below may be duplicates, but just putting them all here as I went through this with @ellemindwave. Looking at the items above, and the ones we noticed while going through the site, it seems that **many** of the issues would be fixed by adjusting the margins (and space the content takes up) on medium screens to be as it is on mobile. 

* Change mobile landscape to small view - topic cards should stretch across full width.
* Iphone 5 feedback page:
![image](https://user-images.githubusercontent.com/1287388/31272385-a247ab7c-aa82-11e7-90bb-554391e4fdda.png)
* Medium sizes - if changing to mobile margins so content is stretched out more horizontally still results in the pros and cons being squished, change to hidden pros/cons on tablet in resource list view
* Ipad button overlapping in questions
![image](https://user-images.githubusercontent.com/1287388/31272714-d4d751fe-aa83-11e7-9856-3e6bc9d5cd01.png). Might be fixed with #519
